test/openmp-cpp-fortran/downstream/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(openmp-cpp-fortran-user LANGUAGES CXX Fortran)
+
+message(STATUS "${BLT_ROOT_DIR}")
```

Comment:
Where does `BLT_ROOT_DIR` come from in this example?

Reply:
This is some cruft left over from debugging, sorry